src/main/webapp/stats.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Stats</title>
+  <link rel="stylesheet" href="/css/main.css">
+  <script>
```

Comment:
You can separate the html and JavaScript codes in this file. You can make a JavaScript file for example stats-loader.js, copy and paste the code between the <script></script> tag in that file and then change the <script> tag in your html file to this(assuming the name of .js file is stats-loader): 
<script src="/js/stats-loader.js"></script>
this way you separate the html and JavaScript code. The html <script> tag shows where to look for the JavaScript file because you have given it the path in src="/js/stats-loader.js", and the stats-loader.js has the functions.